frontend/src/components/Board/Settings/index.tsx

```diff
@@ -278,161 +317,201 @@ const BoardSettings = ({
+                          text="Make votes more significant by limiting them."
+                          title="Limit votes"
+                        >
+                          <Input
+                            css={{ mt: '$8' }}
+                            disabled={!switchesState.maxVotes}
+                            id="maxVotes"
+                            name="maxVotes"
+                            placeholder="Max votes"
+                            type="number"
+                          />
+                        </ConfigurationSwitchSettings>
+                      </>
                     )}
-                </ConfigurationSettings>
 
-                {isSubBoard && (isStakeholderOrAdmin || isOwner || isSAdmin || isResponsible) && (
-                  <TeamResponsibleSettings>
-                    <ConfigurationSwitchSettings
-                      isChecked={switchesState.responsible}
-                      text="Change responsible participant for this board."
-                      title="Team Responsible"
-                      handleCheckedChange={handleResponsibleChange}
-                    >
-                      <Flex
-                        align="center"
-                        css={{
-                          mt: '$10',
-                          opacity: !switchesState.responsible ? '40%' : 'unset',
-                          pointerEvents: !switchesState.responsible ? 'none' : 'unset',
-                          transition: 'all 0.25s ease-in-out',
-                        }}
-                      >
-                        <Text
-                          css={{
-                            mr: '$8',
-                            color: '$primary300',
-                          }}
-                        >
-                          Responsible Lottery
-                        </Text>
-                        <Separator
-                          orientation="vertical"
-                          css={{
-                            '&[data-orientation=vertical]': {
-                              height: '$12',
-                              width: 1,
-                            },
-                          }}
+                    {isRegularBoard &&
+                      (isStakeholderOrAdmin || isOwner || isSAdmin || isResponsible) && (
+                        <ConfigurationSwitchSettings
+                          handleCheckedChange={handleIsPublicChange}
+                          isChecked={switchesState.isPublic}
+                          text="If you make this board public anyone with the link to board can access it. Where to find the link? Just copy the URL of the board itself and share it."
+                          title="Make board public"
                         />
+                      )}
+                  </ConfigurationSettings>
 
+                  {isSubBoard && (isStakeholderOrAdmin || isOwner || isSAdmin || isResponsible) && (
```

Comment:
```suggestion
                  {isSubBoard && (hasPermissions) && (
```

Can you the extract the roles verification to a variable `hasPermissions`? 
`const hasPermissions = isStakeholderOrAdmin || isOwner || isSAdmin || isResponsible`